CLI.md

```diff
@@ -0,0 +1,69 @@
+# xucli
+
+`xucli` is the command line interface that handles much of the basic interaction with a running `xud` instance. If `xud` is installed globally, it can be launched from any directory. In the recommended [xud-docker](User%20Guide.md) setup, it can be used from within the `xud ctl` shell.
+
+To get a list of up-to-date commands, run:
+```
+$ xucli --help
+xucli <command>
+
+Commands:
```

Comment:
I'm not sure if we should include the whole output here since it's subject to change. For example a pending PR will change `channelbalance` to `balance` and we'd have to update this file as well. 🤔